cryptography/src/bls12381/dkg/ops.rs

```diff
@@ -51,14 +51,31 @@ pub fn evaluate_all<V: Variant>(polynomial: &poly::Public<V>, n: u32) -> Vec<V::
     evals
 }
 
+/// A verified commitment.
+pub struct VerifiedCommitment<'a, V: Variant> {
+    commitment: &'a poly::Public<V>,
+}
+
+impl<'a, V: Variant> VerifiedCommitment<'a, V> {
+    /// Create a new verified commitment.
+    fn new(commitment: &'a poly::Public<V>) -> Self {
```

Comment:
Because this is private, it isn't possible to skip verification.

Reply:
Nit: Wonder if it's worth having this function, or just construct it directly within this file